src/machine/behavior/is/other/getab.rs

```diff
@@ -1,6 +1,13 @@
 use machine::state::State;
 
-pub fn getab(_state: &mut State, _x: u8, _y: u8, _z: u8) {
-    unimplemented!();
-}
+pub fn getab(state: &mut State, x: u8, y: u8, z: u8) {
+    // Load operands
+    let ra: u64 = ((y as u64) << 8) + (z as u64);
+    let at: u64 = state.pc.into();
+
+    // Execute
+    let res: u64 = at + 4 * (ra - 2u64.pow(16u32));
```

Comment:
It will even evaluate the `pow()`-call at compile time so there's just a constant left. Use what you think is more readable.